mmaction/core/evaluation/ava_evaluation/np_box_ops.py

```diff
@@ -0,0 +1,102 @@
+# Copyright 2017 The TensorFlow Authors. All Rights Reserved.
```

Comment:
some codes here seem redundant. like `area`

Reply:
Actually not, area is used by iou, iou is used by other codes.